### DISCUSSION TITLE
Change the entire table background color with wrapping Paper element

### DISCUSSION THREAD

**Discussion by amhassen97** [KevinVandy/material-react-table#468] [Q&A]
Hi 

I have a `<Paper />` component wrapping the `<MaterialReactTable />` component, with a global MUI theme. 

I am trying to change the background color of the Paper component as well as the table, however both use the global theme `palette.background.default` value, even when I create a custom `<ThemeProvider />` to wrap the `<Paper />` to override those values.

I am not sure what is causing this, as from my understanding in the docs, the custom theme for the table should override the global theme where the values are different.

Please assist?

Any help is appreciated!

Thanks

**Comment by alessandro-hic** [comment]
I have the same issue, can't set background color when changing theme..
Have you ore somebody got a solution?

**Reply by amhassen97** [reply]
Unfortunately I haven't found a solution yet😕

**Comment by KevinVandy** [comment]
@alessandro-hic @amhassen97 Can you guys share your code because this is supposed to be really simple and just work.

**Comment by KevinVandy** [comment]
V2 has a new `mrtTheme` option where you can set a `baseBackgroundColor`

https://www.material-react-table.com/docs/guides/customize-components#mrt-theme-values
